flowback/ledger/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+from .models import Account, Transaction
+
```

Comment:
If there's no need to use the Serializers outside of the module views themselves, there's no need to declare them seperately in a serializers.py file, it'll improve accessibility to serializers.

Move Serializers to views.py, name them InputSerializer, OutputSerializer, FilterSerializer depending on usecase